neurom/_fst/mm.py

```diff
@@ -273,3 +273,19 @@ def bifurcation_partition(bif_point):
     n = float(sum(1 for _ in ipreorder(bif_point.children[0])))
     m = float(sum(1 for _ in ipreorder(bif_point.children[1])))
     return max(n, m) / min(n, m)
+
+
+def principal_direction_extents(nrn, neurite_type=NeuriteType.all, direction=0):
+    '''Principal direction extent of neurites in neurons'''
+    def _pde(neurite):
```

Comment:
I spent one minute wondering why this is called "Partial Differential Equation" XD